mitodl/mit-open
# News and events for MIT Professional Education

Blocked until MITPE's production API is ready and returning real data

Once the final production api domain is known, that value should be added to `MITPE_BASE_API_URL` in `ol-infrastructure`

Closes https://github.com/mitodl/hq/issues/5090
Closes https://github.com/mitodl/hq/issues/5089

### Description (What does it do?)
Imports news and events from Professional Education API's (official url's for these are still TBD).


### How can this be tested?
- Set `MITPE_BASE_API_URL=https://professional.mit.edu/` in your backend .env file
- Run `./manage.py backpopulate_news_events -p mitpe_news_etl -p mitpe_events_etl`
- It should complete successfully and and import both news and events feed items:
  - http://api.open.odl.local:8063/api/v0/news_events/?feed_type=events
  - http://api.open.odl.local:8063/api/v0/news_events/?feed_type=news
